src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Button from "./Button";
+import "../ButtonPanel.css";
+export default function ButtonPanel() {
+  const btnGroup1 = ["AC", "+/-", "%", "÷"];
+  const btnGroup2 = ["7", "8", "9", "X"];
+  const btnGroup3 = ["4", "5", "6", "-"];
+  const btnGroup4 = ["1", "2", "3", "+"];
+  const btnGroup5 = ["0", ".", "="];
+  return (
+    <div>
+      <div class="buttons">
+        {btnGroup1.map(button => (
```

Comment:
Let us make the appropriate changes to remove these errors

![image](https://user-images.githubusercontent.com/6015315/64136792-c9216980-ce26-11e9-9daa-d60e608835d1.png)